[export] Add mark_compiled_region support

angelayi opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* #149298
* #149297
* __->__ #149296

Differential Revision: [D71298930](https://our.internmc.facebook.com/intern/diff/D71298930)

angelayi committed:
[export] Add mark_compiled_region support

[ghstack-poisoned]

angelayi committed:
Update on "[export] Add mark_compiled_region support"




[ghstack-poisoned]

angelayi commented:
@angelayi has imported this pull request.  If you are a Meta employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D71298930).

angelayi committed:
Update on "[export] Add mark_compiled_region support"


Differential Revision: [D71298930](https://our.internmc.facebook.com/intern/diff/D71298930)

[ghstack-poisoned]

angelayi committed:
Update on "[export] Add mark_compiled_region support"


Differential Revision: [D71298930](https://our.internmc.facebook.com/intern/diff/D71298930)

[ghstack-poisoned]

angelayi commented:
@angelayi has imported this pull request.  If you are a Meta employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D71298930).

zou3519 reviewed (approved):
This is another thing to add to the list of "things export needs Dynamo to do"

angelayi committed:
Update on "[export] Add mark_compiled_region support"


Differential Revision: [D71298930](https://our.internmc.facebook.com/intern/diff/D71298930)

[ghstack-poisoned]

angelayi committed:
Update on "[export] Add mark_compiled_region support"


Differential Revision: [D71298930](https://our.internmc.facebook.com/intern/diff/D71298930)

[ghstack-poisoned]
